packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -1628,10 +1628,10 @@ class ScrollAction extends Action<ScrollIntent> {
     }
   }
 
-  // Find out how much of an increment to move by, taking the different
-  // directions into account.
-  double _getIncrement(ScrollableState state, ScrollIntent intent) {
-    final double increment = _calculateScrollIncrement(state, type: intent.type);
+  /// Find out how much of an increment to move by, taking the different
+  /// directions into account.
+  static double getIncrement(ScrollableState state, ScrollIntent intent) {
```

Comment:
Maybe this could be `getDirectionalIncrement` in the public space? It already call on calculateScrollIncrement. Could you incorporate the logic for a forward intent or translate forward into `intent.direction`?

Reply:
Translated to intent.direction and renamed to getDirectionalIncrement 👍